activities/Chess.activity/js/l10n.js

```diff
@@ -57,7 +57,7 @@ var Localization = {
 
 		// Activity initialization check
 		const SugarActivity = vm.$root.$children.find(function (child) {
-			return child.$options.name == 'SugarActivity';
+			return child.$options.name = 'SugarActivity';
```

Comment:
I'm not comfortable with this fix. From an algorithmic POV, it has no sense to replace a test condition by an assignment!

More globally, because the Chess activity is a Vue.js activity but don't handle localization using `SugarL10n` component, I think the migration for this activity should be a mix of Vue.js migration guide and Vanilla.js migration guide.
It seems that you've just replaced the existing `l10n.js` file by the one provided in the Vanilla.js migration guide and it's not a good idea here.

I think the content of the file should be adapted here.

Reply:
I apologize if my approach caused any confusion. I'll work on adjusting the code to align with both migration guides and find a solution that doesn't involve an assignment operator.